server/coordinator.go

```diff
@@ -386,11 +386,11 @@ func (c *coordinator) removeOperator(op *schedule.Operator) {
 	c.Lock()
 	defer c.Unlock()
 	c.removeOperatorLocked(op)
+	c.limiter.UpdateCounts(c.operators)
```

Comment:
it should move into removeOperatorLocked(), cause other places(such as Line 364) will call removeOperatorLocked directly()